service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -22,143 +34,246 @@
 import static uk.gov.hmcts.reform.fpl.enums.AllocatedJudgeNotificationType.NOTICE_OF_PROCEEDINGS;
 import static uk.gov.hmcts.reform.fpl.enums.AllocatedJudgeNotificationType.SDO;
 
-@ExtendWith(SpringExtension.class)
-public class FeatureToggleServiceTest {
+@ExtendWith(MockitoExtension.class)
+class FeatureToggleServiceTest {
 
     private static final String LD_USER_KEY = "test_key";
 
-    @MockBean
-    private LDClient ldClient;
+    private static LDClient ldClient = Mockito.mock(LDClient.class);
+    private static FeatureToggleService service = new FeatureToggleService(ldClient, LD_USER_KEY);
 
-    private FeatureToggleService featureToggleService;
+    @Captor
+    private ArgumentCaptor<LDUser> ldUser;
 
-    @BeforeEach
-    void setup() {
-        featureToggleService = new FeatureToggleService(ldClient, LD_USER_KEY);
+    @AfterEach
+    void tearDown() {
```

Comment:
I think there could possibly be a better name for this method as it is tearing anything down.
I think we use resetInvocations else where